mep/books/management/commands/export_books.py

```diff
@@ -0,0 +1,111 @@
+'''
+Manage command to export member data for use by others.
+
+Generates a CSV and JSON file including details on every library member in the
+database, with summary details and coordinates for associated addresses.
```

Comment:
think this docblock is left over from copying from the member export :)

Reply:
yes it is, thank you